gatsby-node.ts

```diff
@@ -24,7 +24,7 @@ try {
 
 // ideally problems would be its own query with
 // source nodes: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-node/#sourceNodes
-
+let stream = fs.createWriteStream('ids.log', { flags: 'a' });
```

Comment:
maybe add a comment here explaining what this does / why this is needed?

Reply:
yeah, it's intentional bc `ids.log` is gitignored, so it starts empty at the start of every workflow run :)